lib/Ix/Processor/JMAP.pm

```diff
@@ -37,6 +37,20 @@ has _dbic_handlers => (
     for my $moniker (keys %$source_reg) {
       my $rclass = $source_reg->{$moniker}->result_class;
       next unless $rclass->isa('Ix::DBIC::Result');
+
+      if (
+        $rclass->can('ix_published_method_map')
+        &&
+        (my $method_map = $rclass->ix_published_method_map)
+      ) {
+        for (keys %$method_map) {
+          my $method = $method_map->{$_};
+          $handler{$_} = sub ($self, $ctx, $arg = {}) {
+            $rclass->$method($ctx, $arg);
+          };
+        }
+      }
+
```

Comment:
While this is consistent with the rest of the functions below (ix_get, ix_set, etc...) those functions do all of their work inside of $ctx->txn_do().

It's easy to forget that when writing a custom method for an Ix class. Perhaps this should be nice and wrap any calls in a transaction for safety...